moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -130,53 +139,65 @@ void TrajectoryExecutionManager::initialize()
   }
   catch (pluginlib::PluginlibException& ex)
   {
-    ROS_FATAL_STREAM_NAMED(name_, "Exception while creating controller manager plugin loader: " << ex.what());
+    RCLCPP_FATAL(node_->get_logger(), "Exception while creating controller manager plugin loader: ", ex.what());
```

Comment:
Please add a "LOGGER".

Reply:
You're completely right! ;)

Reply:
For now we just used the names that where already used with MoveIt 1. When restructuring the nodes we will have to address this issue anyway.